includes/admin/class-sensei-learner-management.php

```diff
@@ -380,14 +381,33 @@ public function learners_default_nav() {
 				),
 				admin_url( 'edit.php' )
 			);
-			$title    .= sprintf( '&nbsp;&nbsp;<span class="course-title">&gt;&nbsp;&nbsp;<a href="%s">%s</a></span>', esc_url( $url ), get_the_title( $course_id ) );
+		} else {
+			$back_url = add_query_arg(
+				[
+					'post_type' => 'course',
+					'page'      => 'sensei_learners',
+				],
+				admin_url( 'edit.php' )
+			);
+		}
+
+		$title = '';
+		if ( 0 < $course_id ) {
+			$title .= get_the_title( $course_id );
```

Comment:
Do we need the '.' here?

Reply:
Hmm, I understand why you ask that :)
So I fixed in a bit different way: https://github.com/Automattic/sensei/pull/5025/commits/4668b4b6dabef3107a3075f904f613da26552ce2